MSMS/scripts/extract_DDA.R

```diff
@@ -123,8 +124,8 @@ if (file.size("data_processed/Ingalls_Lab_Standards_MSMS.csv") / 1e6 > 5) {
   stop("Beware! The output file is larger than 5MB - be careful pushing to GitHub.")
 }
 
-library(git2r)
 
+# Automatically commit changes to the Ingalls Standards github
 repo <- repository()
 add(repo, ".gitignore.txt")
```

Comment:
What's the thought behind this line? .gitignore.txt doesn't actually exist as far as I can tell, and I'm nervous auto-adding a file that doesn't exist yet. I don't think the current .gitignore needs to be recommitted every time we update the MSMS data

Reply:
I modified the gitignore to include all the .mzML files, which I think is just good practice since we never want those on the github. I'm seeing the gitignore on both the remote repo on GH and in my own repo, so I'm not sure what you mean by it doesn't exist?